extensions/markdown-language-features/package.json

```diff
@@ -153,6 +174,21 @@
           "when": "editorLangId == markdown && !notebookEditorFocused",
           "group": "navigation"
         },
+        {
+          "command": "markdown.showStaticPreview",
+          "when": "editorLangId == markdown && !notebookEditorFocused",
+          "group": "navigation"
+        },
+        {
+          "command": "markdown.showTextEditor",
+          "when": "!notebookEditorFocused",
```

Comment:
I wonder if there is a context key for which editor type is active, text editor vs markdown preview vs some other type of editor, so these commands can be enabled for exactly the right situation. Do you know @lramos15?

Reply:
This might work https://github.com/microsoft/vscode/issues/113511

Reply:
I just noticed that these conditions don't appear in the `keyboard shortcuts` menu under `when`, and same applies for other options without a default keybinding like `markdown.preview.refresh`. Are they still being applied?

Reply:
I noticed that, when assigning a keybinding to `markdown.showSource` or `markdown.preview.refresh`, they seem to trigger regardless of the conditions set within the extension code? Is there something extra that these commands need in order to follow their respective conditions?

Reply:
update: the `when` fields should now have the commands appear when appropriate